datalad/local/wtf.py

```diff
@@ -138,9 +139,33 @@ def _describe_system():
                                   _t2s(pl.win32_ver())]).rstrip(),
         'max_path_length': get_max_path_length(getpwd()),
         'encoding': get_encoding_info(),
+        'filesystem': {l: _get_fs_type(l, p) for l, p in
+                       [('CWD', os.getcwd()),
+                        ('TMP', tempfile.gettempdir()),
+                        ('HOME', str(Path.home()))]}
     }
 
 
+def _get_fs_type(loc, path):
+    global fs_hint
+    fs_hint = None
+    try:
+        from psutil import disk_partitions
+        match = ""
+        fs = ""
+        for part in disk_partitions():
+            if path.startswith(part.mountpoint) and \
+                    len(match) < len(part.mountpoint):
```

Comment:
such comparison I think could fail to differentiate `/mnt` and `/mntpoint`... since not working with pathlib Paths here to use  smth like `pathlib.Path(part.mountpoint) in pathlib.Path(path).parents` use our  `path_is_subpath`.

```shell
$> python3 -c 'from datalad.utils import path_is_subpath; print(path_is_subpath("/mnt/subpath","/mnt"))'
True
```